### ISSUE TITLE
[BRD] Coffers loot seem incorrect

### ISSUE THREAD

**Issue by BarbzYHOOL** [azerothcore/azerothcore-wotlk#2690]
##### SMALL DESCRIPTION:

In the black vault, the secret safe and the dark coffer do not seem to hold the correct loot tables.

##### CURRENT BEHAVIOUR:

https://wow.gamepedia.com/Secret_Safe this one had 0 loot
https://classic.wowhead.com/object=160845/dark-coffer#comments this one had two loot but it was a backpack and a green pair of boots, seems like it's missing crafting items

##### STEPS TO REPRODUCE THE PROBLEM:

1. .tele blackrockdepths
2. .add 11000
3. .add 11078 12
1. Open all the coffers in the black vault, kill the boss and loot the secret safe (behind the dark coffer) (or just do it as GM mode maybe it works)
1. .add 11197
1. open the dark coffer

##### BRANCH(ES):

master

##### AC HASH/COMMIT:

8760a9f

**Comment by Kitzunu** [comment]
Suggested fix for Secret Safe Entry = 161495
```sql
DELETE FROM `gameobject_loot_template` WHERE `entry` IN (161495);
INSERT INTO `gameobject_loot_template` (`entry`, `item`, `chance`, `questrequired`,  `groupid`, `mincount`, `maxcount`) VALUES
(161495, 22205, 25, 0, 1, 1, 1), -- Secret Safe
(161495, 22255, 25, 0, 1, 1, 1),
(161495, 22256, 25, 0, 1, 1, 1),
(161495, 22254, 25, 0, 1, 1, 1),
(161495, 11309, 100, 1, 1, 1, 1);
```
For dark coffer I hope there is an existing reference entry for it, cuz I am way too lazy to add 170+ entries x))

**Comment by FrancescoBorzi** [comment]
@Kitzunu thanks, can you please [open a PR](http://www.azerothcore.org/wiki/How-to-create-a-PR) with it?

**Comment by Kitzunu** [comment]
ah yea, sure

**Comment by BarbzYHOOL** [comment]
> For dark coffer I hope there is an existing reference entry for it, cuz I am way too lazy to add 170+ entries x))

after studying how the loot system works, I don't see a reason for a reference entry for the loot as it's certainly a unique loot template never found anywhere else lol

**Comment by BarbzYHOOL** [comment]
no, in fact it can possibly exist a reference entry unused because it's used for bosses that drop multiple items from a referenced loot table (example: kel thuzad)


So I checked for the first loot in the list to see if there was a ref entry and sadly, not:

```sql
SELECT * FROM `reference_loot_template` WHERE `Item` = '11752' OR `Item` = 11751
```

We need a way to web scrap all the items ID, drop chance etc... at once

**Comment by BarbzYHOOL** [comment]
https://github.com/swimmesberger/WoWHead-Item-Parser might be useful

**Comment by Helias** [comment]
@BarbzYHOOL  "WoWHead-PHP-Extractors" is a fork of my repo xD
That project is old, I don't think it still work, I would make a new one but I am busy with AzerothCore and Keira3 :sweat_smile:
Anyway several times WoWHead give random drop chance, I made the wowhead loot extractor mainly to check if a loot is present on WoWHead and we have not it, so we are sure that we need to implement that loot.
Probably this last function still works.

Anyway I wrote that software long time ago, today I would rewrite it in Python+Selenium or PuppeteerJS.
